CONTRIBUTING.md

```diff
@@ -8,9 +8,11 @@ Note: New contributors should first contact [Ben Elan](mailto:belan@esri.com) or
 
 Calcite Components is still in its early stages. You can help most by:
 
-- Adding ideas for components to our [wishlist project](https://github.com/Esri/calcite-components/projects/2). We are using the wishlist to gather ideas about useful components which we may be able to add to Calcite Components.
-- Working on [the issues](https://github.com/Esri/calcite-components/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee) marked as `help wanted`. There is also a `good first issue` label if you are just getting started.
-- If you want to help develop components take a look at [1.0.0 components](https://github.com/Esri/calcite-components/projects/1) which are the components we are targeting for the first release of Calcite Components. Before starting development please review our [component conventions](./conventions/README.md) and the [Stencil documentation](https://stenciljs.com/docs/host-element).
+- Adding ideas for components by [creating a New Component issue](https://github.com/Esri/calcite-components/issues/new?assignees=&labels=new+component%2C0+-+new%2Cneeds+triage&template=new-component.yml).
+- Requesting features for existing components by [creating a Enhancement issue](https://github.com/Esri/calcite-components/issues/new?assignees=&labels=enhancement%2C0+-+new%2Cneeds+triage&template=enhancement.yml).
+- Reporting problems by [creating a Bug issue](https://github.com/Esri/calcite-components/issues/new?assignees=&labels=bug%2C0+-+new%2Cneeds+triage&template=bug.yml).
+- Working on [the issues marked as `help wanted`](https://github.com/Esri/calcite-components/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee). There is also a `good first issue` label if you are just getting started. Before working on an issue, make sure to comment on it and ask for the action items. Sometimes there is additional context needed that is not specified in the issue.
```

Comment:
Since there is an action requested with the statement, could we add a sub-bullet for the "Before working on an issue..." sentence?

Also, added a suggestion for the action's clarity below.

Perhaps:
- Working on the issues marked as help wanted. There is also a good first issue label if you are just getting started.
    - Before working ~on an issue, make sure to~ comment on [the issue] ~~it~~ and ask for the action items. Sometimes there is additional context needed that is not specified in the issue.